gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ActiveClusterMonitor.java

```diff
@@ -79,6 +79,10 @@ public void start()
                             List<ClusterStats> stats = new ArrayList<>();
                             for (Future<ClusterStats> clusterStatsFuture : futures) {
                                 ClusterStats clusterStats = clusterStatsFuture.get();
+                                if (!clusterStats.healthy()) {
+                                    log.error("cluster %s is unhealthy", clusterStats.clusterId());
+                                    log.debug("cluster stats: %s", clusterStats);
```

Comment:
Will this print the stats, or just the default `Object:toString`?

Reply:
on a second thought, I'm not sure whether this will add more value, or creating another observer will. I'm thinking we can create a new observer to just alert when no clusters are available instead of printing each cluster's stats. WDYT